tf2onnx/rewriter/rnn_utils.py

```diff
@@ -40,37 +40,26 @@ def __init__(self, kernel, bias, forget_bias):
         self.forget_bias = forget_bias
 
 
-class RnnInitializers:
-    def __init__(self, c_init, h_init, c_h_shared_init):
-        self.c_init_input_id = None
-        self.h_init_input_id = None
-        self.share_init_node = None
-        self.share_init_input_id = None
-
-        if c_h_shared_init:
-            self.share_init_input_id = c_h_shared_init
-            self.share_init_node = True
-        else:
-            self.c_init_input_id = c_init
-            self.h_init_input_id = h_init
-            self.share_init_node = False
-
-
 class RnnProperties:
     def __init__(self):
         # RNN input who are outside of rnn scope
+        self.input_x = None  # InputTensorArray

```

Comment:
if you would like, you can just give up using RnnProperties, just define UnitRnnContext inheriting Context, and put all those properties in the context. Then next step, we can just remove this data structure along with old lstm rewriter code.

Reply:
sure!